fastapi-challenges/jwt-auth/devashmit/auth.py

```
@@ -0,0 +1,48 @@
+"""
+JWT creation and verification utilities.
+"""
+
+from datetime import datetime, timedelta, timezone
+
+from jose import JWTError, jwt
+from passlib.context import CryptContext
+
+# Change this to a long random secret in production — never hardcode in real apps
+SECRET_KEY = "change-me-to-a-long-random-secret-in-production"
```

Ruff (0.15.14)

[error] Possible hardcoded password assigned to: "SECRET_KEY"

(S105)